node_modules/oae-authentication/lib/strategies/shibboleth/init.js

```diff
@@ -84,6 +84,11 @@ module.exports = function(config) {
                 'authoritative': true
             };
 
+            // Set users whose name resembles a Shibboleth identifier as private (eg. starts with http://)
+            if (displayName.indexOf('http://') === 0 || displayName.indexOf('shibboleth!') != -1) {
+                opts.visibility ='private';
+            } 
+
```

Comment:
If I understand correctly, that widget is in 3akai-ux, so do I need to make a pull request there if I need to make changes to it?